parser/parser.y

```diff
@@ -5221,6 +5230,53 @@ AdminStmt:
 			JobIDs: $6.([]int64),
 		}
 	}
+|	"ADMIN" "SHOW" "LOG" AdminShowLog
```

Comment:
"LOG" is too general, "SLOW_QUERY" is better.

Reply:
@winkyao What's your opinion ?

Reply:
I vote for the `slow query`.

Reply:
Support for using  `"LOG"` .

Reply:
How about `ADMIN SHOW SLOW`?  I agree that log is too generic, but with `slow` it could be implied.